data-tx-hibernate6/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id "io.micronaut.build.internal.data-module"
+}
+
+dependencies {
+    annotationProcessor mn.micronaut.inject.java
+    annotationProcessor mn.micronaut.graal
+
+    api projects.dataTx
+    api (libs.hibernate6.core) {
+        exclude group: 'org.jboss.spec.javax.transaction', module: 'jboss-transaction-api_1.2_spec'
+        exclude group: 'org.javassist', module: 'javassist'
+        exclude group: 'org.jboss:jandex', module: 'jar'
+    }
+
+    // TODO: Import hibernate6 when ready
+    api (mnSql.micronaut.hibernate.jpa) {
```

Comment:
This will be new module in micronaut-sql if this is merged and will be like `mnSql.micronaut.hibernate6.jpa`

Reply:
Done